src/meta/meta_bulk_load_service.cpp

```diff
@@ -1654,6 +1679,7 @@ void bulk_load_service::do_continue_app_bulk_load(
     {
         zauto_write_lock l(_lock);
         _apps_in_progress_count[app_id] = in_progress_partition_count;
+        _apps_failover_count[app_id] = 0;
```

Comment:
Why did you set _apps_failover_count[app_id] to 0 when you continue to bulk load? I think you can reserve the old value.

Reply:
`do_continue_app_bulk_load` is called when meta server crash and restart, find bulk load context on zk and continue to do bulk load. In this case, `_apps_failover_count[app_id]` should be initialized as 0.